app/src/docs/_examples/dropdownMenu/Basic.example.tsx

```diff
@@ -107,7 +108,15 @@ export default function BasicDropdownMenuExample() {
                 closeOnMouseLeave="boundary"
                 closeDelay={ 400 }
                 renderBody={ (props) => renderDropdownBody(props) }
-                renderTarget={ (props) => <Avatar img="https://avatars.dicebear.com/api/human/avatar12.svg?background=%23EBEDF5&radius=50" size="36" { ...props } /> }
+                renderTarget={ (props) => (
+                    <LinkButton
+                        icon={ PersonIcon }
+                        caption="Alex Smith"
+                        size="36"
+                        { ...props }
+                        rawProps={ { 'aria-haspopup': props.isDropdown, 'aria-expanded': props.isOpen } }
```

Comment:
Here I added a couple of `aria` labels for proper screen reader announcing. Probably it will be better if `Dropdown` component can pass them via `props.rawProps` for example .